<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependencyInstallationDialog</class>
 <widget class="QDialog" name="DependencyInstallationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Site Schedule Optimization</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Python Dependency Installation</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textDisplay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Site Schedule Optimization has detected its python dependencies are either not installed or out-of-date. They are required for operation. 

They are installed using Pip (Package Installer for Python) into the plugin directory so your system python installation will not be affected.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="installButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> Install</string>
   </property>
   <property name="icon">
    <iconset theme="software-update-available">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> Cancel</string>
   </property>
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="installingLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Installing...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
